sdhook.go

```diff
@@ -291,3 +291,17 @@ func (sh *StackdriverHook) buildErrorReportingEvent(entry *logrus.Entry, labels
 	}
 	return errorEvent
 }
+
+func severityString(l logrus.Level) string {
```

Comment:
Please convert this func to a `switch` or a map lookup. Otherwise, happy to merge.

Reply:
Sorry for the delay, but I have updated this for you! THanks. :)